### PR TITLE
chore: clean up sdk metadata code

### PR DESCRIPTION
**Pull request by deekshas8**
Closes SAP/cloud-sdk-backlog#620.

### CODE REVIEW

File: packages/generator-common/package.json [hunk 1]
```diff
@@ -37,15 +37,13 @@
     "readme": "ts-node ../../scripts/replace-common-readme.ts"
   },
   "dependencies": {
-    "@sap-cloud-sdk/util": "^2.8.0",
```
Comment (jjtang1985) [inline packages/generator-common/package.json:40]: [req] Perhaps this is not intended after the release?

File: test-packages/e2e-tests/package.json [hunk 2]
```diff
@@ -57,7 +55,9 @@
     "requires": {
       "db": {
         "kind": "sqlite",
-        "model": "*"
```
Comment (jjtang1985) [inline test-packages/e2e-tests/package.json:60]: [q] could you please share the reason?

**Review by jjtang1985 — CHANGES_REQUESTED** [review]
LGTM.
Please check the old SDK version used.

File: test-packages/e2e-tests/package.json (hunk 2 above)
Comment (deekshas8) [reply test-packages/e2e-tests/package.json:60]: This change was unintended, will revert it.

File: packages/generator-common/package.json (hunk 1 above)
Comment (deekshas8) [reply packages/generator-common/package.json:40]: You're right, I reverted it.

**Review by jjtang1985 — APPROVED** [review]

**Pull request merged**